package.json

```diff
@@ -11,14 +11,35 @@
   "scripts": {
     "test": "grunt test lint",
     "start": "node ./bin/www",
-    "preinstall": "node version-check.js"
+    "preinstall": "node version-check.js",
+    "eslint": "eslint web/js/**/wv.*.js test/**/*.js",
+    "stylefmt": "stylelint web/css/*.css --fix",
+    "watch": "grunt watch",
+    "autoprefix": "postcss web/css/*.css --use autoprefixer -d web/css/ --no-map",
+    "build": "grunt build",
+    "config": "grunt config",
+    "fetch": "grunt fetch",
+    "site": "grunt site",
+    "stylelint": "stylelint web/css/*.css",
+    "rpm-only": "grunt rpm-only",
+    "apache-config": "grunt apache-config",
+    "update": "grunt update",
+    "update-packages": "grunt update-packages",
+    "check": "grunt check",
+    "distclean": "grunt distclean",
+    "lint": "grunt lint",
+    "chrome-tests": "grunt chrome-tests",
+    "firefox-tests": "grunt firefox-tests",
+    "e2e": "grunt e2e",
+    "default": "grunt default"
   },
   "devDependencies": {
     "autoprefixer": "7.1.1",
     "babel-polyfill": "6.16.0",
     "bluebird": "3.4.6",
     "buster": "0.7.12",
     "chromedriver": "2.25.1",
+    "cssnano": "^4.0.0-rc.2",
```

Comment:
Where is this used?

Reply:
No where yet. It is intended to replace `grunt-contrib-cssmin`